tensorflow/training/docker/1.15/py3/cu110/Dockerfile.gpu

```diff
@@ -189,12 +189,12 @@ RUN ${PIP} install --no-cache-dir -U \
 
 # Install extra packages
 RUN pip install --no-cache-dir -U \
-    bokeh \
-    imageio \
-    plotly \
-    seaborn \
-    shap
-
+   "bokeh>=2.3,<3" \
+   "imageio>=2.9,<3" \
+   "opencv-python>=4.3,<5" \
```

Comment:
Please delete `opencv-python` from here, as it is already installed in line 176.

Reply:
I deleted from 176